client/js/lounge.js

```diff
@@ -1107,16 +1108,26 @@ $(function() {
 	});
 
 	chat.on("input", ".search", function() {
-		var value = $(this).val().toLowerCase();
-		var names = $(this).closest(".users").find(".names");
-		names.find(".user").each(function() {
-			var btn = $(this);
-			var name = btn.text().toLowerCase().replace(/[+%@~]/, "");
-			if (name.indexOf(value) === 0) {
-				btn.show();
-			} else {
-				btn.hide();
-			}
+		const value = $(this).val().toLowerCase();
+		const names = $(this).closest(".users").find(".names");
+
+		names.find(".user").each((i, el) => {
+			$(el).text($(el).text().replace(/<\/?b>;/, "")).hide();
+		});
+
+		const fuzzyOptions = {
+			pre: "<b>",
+			post: "</b>",
+			extract: el => $(el).text().toLowerCase().replace(/[+%@~]/, "")
```

Comment:
Do you really need the `replace` when you're fuzzy searching?

Also `fuzzy` has `caseSensitive`, which does not appear to be on by default.